db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_09_12_172742) do
+ActiveRecord::Schema.define(version: 2019_08_16_120445) do
```

Comment:
Maybe a relic of undoing migrations in this branch? Probably don't need it?

Reply:
Yep, now that I have merged in the migration from `master` -> `team-z/facilities-datasource-switchover` -> here, this is gone.